src/flowgraph/FlowGraphNode.h

```diff
@@ -40,8 +40,16 @@
 
 // Set this to 1 if using it inside the Android framework.
 // This code is kept here so that it can be moved easily between Oboe and AAudio.
+#ifndef FLOWGRAPH_ANDROID_INTERNAL
 #define FLOWGRAPH_ANDROID_INTERNAL 0
+#endif
 
+// Set this to a name that will prevent AAudio from calling into Oboe.
```

Comment:
I think this needs a clearer description/comment since most developers won't know what this refers to.

Reply:
OK